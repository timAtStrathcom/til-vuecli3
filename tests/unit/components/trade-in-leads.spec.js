import { shallowMount } from '@vue/test-utils'
import TradeInLeads from '@/components/trade-in-leads'

describe('trade-in-leads', () => {
  it('matches snapshot', () => {
    const tradeInLeads = shallowMount(TradeInLeads)
    expect(tradeInLeads).toMatchSnapshot()
  })
})
