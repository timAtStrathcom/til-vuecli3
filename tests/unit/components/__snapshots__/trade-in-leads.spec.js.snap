// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trade-in-leads matches snapshot 1`] = `
<div class="trade-in-widget box">
  <div class="title">
    Value Your Trade
    <img src="@/assets/img/black-book-logo--grey.png" class="cbb-logo is-pulled-right">
  </div>
  <tradeinautocomplete-stub value=""></tradeinautocomplete-stub>
</div>
`;
